Modern/ops/src/test/java/org/bonej/ops/ellipsoid/DistanceFromEllipsoidSurfaceOpTest.java

```diff
@@ -0,0 +1,147 @@
+package org.bonej.ops.ellipsoid;
+
+import net.imagej.ImageJ;
+import net.imagej.ops.special.function.BinaryFunctionOp;
+import net.imagej.ops.special.function.Functions;
+import net.imglib2.type.numeric.real.DoubleType;
+import org.apache.commons.math3.random.UnitSphereRandomVectorGenerator;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.scijava.vecmath.Matrix3d;
+import org.scijava.vecmath.Point3d;
+import org.scijava.vecmath.Vector3d;
+
+
+import java.util.function.Supplier;
+
+import static org.junit.Assert.assertEquals;
+
+
+/**
+ * Tests for {@link DistanceFromEllipsoidSurfaceOp}.
+ *
+ * @author Alessandro Felder
+ */
+public class DistanceFromEllipsoidSurfaceOpTest {
+
+    private static final ImageJ IMAGE_J = new ImageJ();
+    private static BinaryFunctionOp<Ellipsoid, Point3d, DoubleType> distanceFromEllipsoidSurfaceOp;
+    private static UnitSphereRandomVectorGenerator sphereRng = new UnitSphereRandomVectorGenerator(3);
+
+    @BeforeClass
+    public static void setUpBeforeClass() throws Exception {
+        distanceFromEllipsoidSurfaceOp = Functions.binary(IMAGE_J.op(), DistanceFromEllipsoidSurfaceOp.class, DoubleType.class, Ellipsoid.class, Point3d.class);
+    }
+
+    @Test
+    public void testDistanceOnAxesForAxisAlignedEllipsoid(){
+
+        Ellipsoid ellipsoid = new Ellipsoid(1.0, 2.0, 3.0);
+
+        Point3d outsidePoint  = new Point3d(2.0,0.0,0.0);
+        Point3d insidePoint  = new Point3d(0.0,1.0,0.0);
+        Point3d surfacePoint  = new Point3d(0.0,0.0,3.0);
+
+        double distanceToOutsidePoint = distanceFromEllipsoidSurfaceOp.calculate(ellipsoid,outsidePoint).get();
+        double distanceToInsidePoint = distanceFromEllipsoidSurfaceOp.calculate(ellipsoid,insidePoint).get();
+        double distanceToSurfacePoint = distanceFromEllipsoidSurfaceOp.calculate(ellipsoid,surfacePoint).get();
+
+        assertEquals("Distance to outside point failed",1.0,distanceToOutsidePoint,1.0e-12);
+        assertEquals("Distance to inside point failed",1.0,distanceToInsidePoint,1.0e-12);
+        assertEquals("Distance to surface point failed",0.0,distanceToSurfacePoint,1.0e-12);
+
```

Comment:
Three tests in one method:
testOutsidePoint
testInsidePoint
testSurfacePoint
If these were separate, it'd be quicker to figure out what doesn't work if test(s) fail